finance_manager/src/main/resources/db/changelog/changeset/create-table-categories.sql

```diff
@@ -0,0 +1,22 @@
+--liquibase formatted sql
+--changeset "gs":5
+
+CREATE TABLE IF NOT EXISTS category_types
+(
+    ct_id    bigserial PRIMARY KEY,
+    ct_types varchar(255) NOT NULL
+);
+
+CREATE TABLE categories

```

Comment:
Let's add preconditions `IF NOT EXISTS` to all creating table scrpits

Reply:
Done